DSharpPlus.UnifiedCommands/Message/IErrorHandler.cs

```diff
@@ -0,0 +1,25 @@
+using DSharpPlus.Entities;
+
+namespace DSharpPlus.UnifiedCommands.Message;
+
+/// <summary>
+/// A interface used in dependency injection to set the default error handler.
+/// </summary>
+public interface IErrorHandler
```

Comment:
are we sure this should be in .Message? shouldn't the error handler be in the root namespace, in case we want to use it from other parts?

Reply:
If we want IErrorHandler to be extendable so others can implement their own errors and be supported in application commands as well yes. And that seems to be the plan.